src/components/Game.js

```diff
@@ -935,24 +880,11 @@ class Game extends React.Component{
         // this.state.round = 4;
         this.setState({round: 0})
         // this.render();
+        let a = console.log(document.getElementById('teams'))
```

Comment:
what is `a` for?

Reply:
check if this div tag exists, if not, then I cannot modify his attibute e.g. visibility, width, heights. Just for debugging, you can delete it.

Reply:
I mean should it be `console.log(document.getElementById('teams'))` ?

Reply:
I can type ```a``` while in debugging console, which is easier then. But that's true.

Reply:
Got it. THx